### PR TITLE
Prevent head request from being done on disabled domains

### PR DESCRIPTION
**Pull request by TiddoLangerak**
@Magnetme/developers - Ready for review!

### CODE REVIEW

File: build/angular-retina.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 * https://github.com/jrief/angular-retina
-* Copyright (c) 2014 Jacob Rief; Licensed MIT */
+* Copyright (c) 2015 Jacob Rief; Licensed MIT */
 // Add support for Retina displays when using element attribute "ng-src".
 // This module overrides the built-in directive "ng-src" with one which
 // distinguishes between standard or high-resolution (Retina) displays.
 (function (angular, undefined) {
   'use strict';
-  var infix = '@2x', data_url_regex = /^data:([a-z]+\/[a-z]+(;[a-z\-]+\=[a-z\-]+)?)?(;base64)?,(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$/i;
+  var infix = '@2x', data_url_regex = /^data:([a-z]+\/[a-z]+(;[a-z\-]+\=[a-z\-]+)?)?(;base64)?,(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$/, pattern = /.*/;
```
Comment (wouterwillems10) [inline build/angular-retina.js]: can you explain this change?

File: lib/angular-retina.js [hunk 2]
```diff
@@ -61,7 +57,12 @@ ngRetina.directive('ngSrc', ['$window', '$http', function($window, $http) {
     }
 
     function set2xVariant(img_url) {
+      if(pattern.test(img_url)) {
+        setImgSrc(img_url);
+        return;
+      }
```
Comment (wouterwillems10) [inline lib/angular-retina.js]: why is this piece of code here as well as in the other file?

File: build/angular-retina.js (hunk 1 above)
Comment (TiddoLangerak) [inline build/angular-retina.js]: This is the build, I think I forgot to push the build last time.

File: lib/angular-retina.js (hunk 2 above)
Comment (TiddoLangerak) [inline lib/angular-retina.js]: The other file is the build.

**Comment by wouterwillems10** [comment]
:+1:

File: lib/angular-retina.js (hunk 2 above)
Comment (TiddoLangerak) [inline lib/angular-retina.js]: I missed the negation here at line 60, fix has been pushed to master directly.

**Pull request merged**